HealthcareBase/Service/MedicationService/MedicationService.cs

```diff
@@ -24,9 +26,33 @@ public Medication GetByID(int id)
             return medicationRepository.Repository.GetByID(id);
         }
 
-        public IEnumerable<Medication> GetAll()
+        public IEnumerable<MedicationDto> GetAll()
```

Comment:
This method should be private because it doesn't return valid information ( Quantity will always be 0 )

Reply:
Fixed.